.github/workflows/secretlint.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node_version: 16
+          node-version: 16
```

Comment:
setup-node action.yml says
https://github.com/actions/setup-node/blob/bed538bd04a773f1616efba082915a7638d8eb5f/action.yml#L8
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043844626

Reply:
I already fixed it, #6